security/rbac/scripts/enable-rbac-connect.sh

```diff
@@ -0,0 +1,152 @@
+confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+
+echo -e "\n# List the role bindings for the principal User:$ADMIN_CONNECT for the Connect cluster"
+echo "confluent iam rolebinding list --principal User:$ADMIN_CONNECT --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID"
+confluent iam rolebinding list --principal User:$ADMIN_CONNECT --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+
+echo -e "\n# Grant the principal User:connector to the DeveloperManage role for Connector:$CONNECTOR_NAME"
+echo "confluent iam rolebinding create --principal User:connector --role DeveloperManage --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID"
+confluent iam rolebinding create --principal User:connector --role DeveloperManage --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+
+echo -e "\n# Grant the principal User:connector to the DeveloperWrite role for Connector:$CONNECTOR_NAME"
+echo "confluent iam rolebinding create --principal User:connector --role DeveloperWrite --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID"
+confluent iam rolebinding create --principal User:connector --role DeveloperWrite --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+
+echo -e "\n# Grant the principal User:connector to the ResourceOwner role for Topic:$DATA_TOPIC"
+echo "confluent iam rolebinding create --principal User:connector --role ResourceOwner --resource Topic:$DATA_TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:connector --role ResourceOwner --resource Topic:$DATA_TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# List the role bindings for the principal User:connector for the Kafka cluster"
+echo "confluent iam rolebinding list --principal User:connector --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding list --principal User:connector --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# List the role bindings for the principal User:connector for the Connect cluster"
+echo "confluent iam rolebinding list --principal User:connector --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID"
+confluent iam rolebinding list --principal User:connector --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+
+get_cluster_id_schema_registry
+echo -e "\n# Grant the principal User:$CLIENT to the ResourceOwner role for Subject:${DATA_TOPIC}-value"
```

Comment:
Should be `User:connector` instead of `User:$CLIENT` (left an in-depth explanation as a comment on the `submit_datagen_pageviews_config_avro.sh` file).

Reply:
I'm changing everything from `connector` to `client`.